plugins/cliconf/iosxr.py

```diff
@@ -46,6 +46,22 @@
     - name: ANSIBLE_IOSXR_COMMIT_CONFIRMED_TIMEOUT
     vars:
     - name: ansible_iosxr_commit_confirmed_timeout
+  commit_confirmed_label:
+    type: str
+    description:
+    - Adds label to commit confirmed.
+    env:
+    - name: ANSIBLE_IOSXR_COMMIT_CONFIRMED_LABEL
+    vars:
+    - name: ansible_iosxr_commit_confirmed_label
+  commit_confirmed_comment:
+    type: str
+    description:
+    - Adds comment to commit confirmed..
+    env:
+    - name: ANSIBLE_IOSXR_COMMIT_CONFIRMED_COMMENT
+    vars:
+    - name: ansible_iosxr_commit_confirmed_comment
```

Comment:
```suggestion
  commit_label:
    type: str
    description:
    - Adds label to commit.
    env:
    - name: ANSIBLE_IOSXR_COMMIT_LABEL
    vars:
    - name: ansible_iosxr_commit_label
  commit_comment:
    type: str
    description:
    - Adds comment to commit.
    env:
    - name: ANSIBLE_IOSXR_COMMIT_COMMENT
    vars:
    - name: ansible_iosxr_commit_comment
```

Reply:
@Qalthos I am not sure if we want to call that just because it associates with the commit command. Primarily this is commit confirmed functionality.